classes/class4/hw/public/javascripts/main.js

```diff
@@ -0,0 +1,33 @@
+var $form1 = $("#addForm");
+
+var onSuccess1 = function(data, status){
+	console.log('main success');
+	console.log(data);
+	var out = "<form id= 'ajax-form1' action='outOfStock' method= 'POST'>" +
+			"<input type='text' name='id' value = "+data._id+" readonly/><br/>"+
+			"<input type='text' name='name' value='"+data.name+"'/><br/>" +
+  			"<input type='text' name='price' value='"+data.price+"'/><br/>"+
+  			"<input type ='submit' value='Edit' formaction = 'edit'>"+
+			"<input type ='submit' value='Out of Stock'>" +
+			"</form>";
+	$("#result").html(out);
```

Comment:
You probably want to append new ingredients rather than setting the contents of a div to it.